Dockerfile

```diff
@@ -22,5 +23,9 @@ RUN ln -s /testdata src/test/resources
 # Create a simple unit test runner
 ENV CI true
 RUN echo "cd /gatk/ && ./gradlew test" >/root/run_unit_tests.sh
+RUN echo "./gradlew test" >/gatk/run_unit_tests.sh
```

Comment:
This should also be removed -- this will just get overwritten by the cp command later